packages/ilios-common/package.json

```diff
@@ -102,6 +102,7 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-load-initializers": "^2.1.2",
+    "ember-local-storage": "^2.0.7",
```

Comment:
Is this used somewhere? I can't find it.

Reply:
ah damn, i _knew_ i forgot something. this was added to make the filter selection stick. but i haven't rigged this up yet. 

i'm taking this back to the workbench...

Reply:
implemented in https://github.com/ilios/frontend/pull/7786/commits/407a4521cc9783133ec80a434862a22213a3b6b0

see https://www.npmjs.com/package/ember-local-storage for plugin details.